src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -32,6 +32,8 @@
     <Configurations>Debug;Release</Configurations>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <NoWarn>SA1123</NoWarn>
+    <MDocDocumentationDirectory>$(MSBuildThisFileDirectory)..\docs\en</MDocDocumentationDirectory>
```

Comment:
we can move all those changes to [here](https://github.com/xamarin/XamarinCommunityToolkit/blob/develop/src/Directory.Build.props) and avoid duplication between Markup and XCT projects

Reply:
Haha yeah that was what I meant by more optimization. But ok, ok.... You're so tough! I will do it right now which is better anyway :D